featuretools/tests/primitive_tests/test_max_consecutive.py

```diff
@@ -126,7 +126,7 @@ def test_all_int(self, dtype):
 
     def test_all_float(self, dtype):
         if dtype == "int64":
-            pytest.skip("floats not supported in int64")
+            pytest.skip("test array contains floats which are not supported int64")
```

Comment:
Nitpick: the floats used in this test could all be represented as integers. Should we change?